cmd/tools/grafana/grafana.go

```diff
@@ -433,7 +433,36 @@ func newLabelVar(label string) []byte {
   "skipUrlSync": false,
   "sort": 0,
   "type": "query"
-}`, label, cases.Title(language.Und).String(label), label)
+}`, label, title, label)
+}
+
+func newClusterLabelVar(varName string, label string) []byte {
```

Comment:
If user is using prefix feature in Harvest then this hardcoding of metric name `cluster_new_status` will not work.

Reply:
Tried both ways, 
```
harvest % bin/grafana import --add-cluster-label netapp_cluster -p  ka
```
and 
```
harvest % bin/grafana import -p ka --add-cluster-label netapp_cluster
```

It's handling prefix for this new var as well
<img width="1670" height="475" alt="image" src="https://github.com/user-attachments/assets/3d9fb0b6-8aaf-4dd6-8d93-69f58c0519c4" />